src/components/Text/Text.tsx

```diff
@@ -57,18 +57,21 @@ export type Props = {
  * `import {Text} from "@chanzuckerberg/eds";`
  *
  * There are two perceived use cases for the text component.
- * One is to decorate <p> and <span> with thematic variants.
- * Defaults to <p> and should pass as="span" to set as <span>
+ * One is to decorate `<p>` and `<span>` with thematic variants.
+ * Defaults to `<p>` and should pass `as="span"` to set as `<span>`.
```

Comment:
Before:
<img width="1360" alt="text component in storybook before this change" src="https://user-images.githubusercontent.com/7761701/200638042-6902fc15-2b54-4cb6-b1f3-1ecdf207d203.png">

After:
<img width="1360" alt="text component in storybook after this change" src="https://user-images.githubusercontent.com/7761701/200638063-1bd2fce0-3c21-4813-934c-5cf214754807.png">